opencga-app/app/scripts/azure/README.md

```diff
@@ -202,9 +205,11 @@ Link VCF genome file to your newly created study
 sudo /opt/opencga/bin/opencga.sh files link -i ALL.chr22.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz -s 1kG_phase3
 ```
 
+> Note: You can complete the following two steps as one by removing `--transform` from the command. If you're running in the Azure ARM version this will make it easier as the process will be queued via Azure Batch. 
+
 Transform file (view progress in separate daemon terminal)
 ```
-sudo /opt/opencga/bin/opencga.sh variant index --file ALL.chr22.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz --transform -o outDir
+sudo /opt/opencga/bin/opencga.sh variant index --file v2ALL.chr22.phase3_shapeit2_mvncall_integrated_v5a.20130502.genotypes.vcf.gz --transform -o outDir
```

Comment:
is this intentional? doesn't look like the linked file has this prefix

Reply:
Good stop this is a hangover from my working around and issue. Will fix up.